day_6/exercises/burrito.js

```diff
@@ -16,6 +16,15 @@ class Burrito {
     this.base = base;
     this.toppings = toppings;
   }
+  addTopping (topping){
+    this.toppings = topping;
```

Comment:
It looks more like you are reassigning all of the topping rather than adding one. Remember- your toppings are an array, so look into methods that will help you add something additional to it.